app/views/spree/admin/gdpr_requests/_form.html.erb

```diff
@@ -0,0 +1,30 @@
+<div data-hook="admin_gdpr_request_form_fields">
+  <div class="row">
+    <div class="left col-12" data-hook="admin_gdpr_request_form">
+      <div class="row">
+        <div class="col-6" data-hook="admin_gdpr_request_form_user">
+          <%= f.field_container :user do %>
+            <%= f.label :user_id, Spree::User.model_name.human %><br />
+            <%= f.collection_select(:user_id, @users, :id, :email, { include_blank: t('spree.match_choices.none') }, { class: 'custom-select fullwidth' }) %>
```

Comment:
This could be a problem on large stores with tens of thousands of users. Is there a way to perform an AJAX search instead? Maybe you can look at the order form for inspiration.